src/octree/generation.rs

```diff
@@ -159,7 +159,7 @@ fn split_node<'a, P>(
     scope: &Scope<'a>,
     octree_data_provider: &'a OnDiskDataProvider,
     octree_meta: &'a octree::OctreeMeta,
-    query_attributes: &'a HashMap<String, AttributeDataType>,
+    attribute_data_types: &'a HashMap<String, AttributeDataType>,
```

Comment:
nit: in point_cloud/generation this very Hashmap is called just attributes or attributes_with_xyz

Reply:
Yes, then we should replace it there as well, but this is another repo...

Reply:
Actually here we really have attributes without xyz.